src/model.js

```diff
@@ -14,4 +14,44 @@ function getRandomIndex(array) {
 
 export function getRandomUser(users) {
   return users[getRandomIndex(users)];
+}
+
+export function getAverageWater(hydrationData, id) {
+  const userData = hydrationData.filter(data => data.userID === id); 
+  
```

Comment:
Is there a way we could DRY up the filter method by adding another parameter and adjusting the getUserData function, or perhaps it's own function? You're filtering for the same thing on lines 20 and 40.